### ISSUE TITLE
Fix S1450: false positive on expression body property

### ISSUE THREAD

**Issue by sm-g** [SonarSource/sonar-csharp#1820]
```cs
private string _entityString;
public string EntityString => _entityString ?? (_entityString = "value");
```

7.4.0.6452
VS 15.8.1

**Comment by Evangelink** [comment]
Hi @sm-g,

This is indeed a FP, thanks for the feedback.

Cheers,
Amaury
